src/DemandActionMenu.ts

```diff
@@ -0,0 +1,66 @@
+import { html, css, LitElement } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { action } from './priv.js';
+
+import './ActionItem.js';
+import './TestElement.js';
+
+/**
+ * Grid menu of demand action types
+ */
+@customElement('demand-action-menu')
+export class DemandActionMenu extends LitElement {
+  @property({ type: String }) title = 'My privacy request';
```

Comment:
You should integrate [localization](https://lit.dev/docs/localization/overview/) into the design as soon as possible.